db/migrate/20250207095204_remove_finished_column_from_projects.rb

```diff
@@ -0,0 +1,19 @@
+class RemoveFinishedColumnFromProjects < ActiveRecord::Migration[7.1]
+  def up
+    Project.all.each do |project|
+      if project.school_project.present?
+        project.school_project.update!(finished: project.finished)
```

Comment:
`project.finished` hasn't been used anywhere yet, so you shouldn't need this

Reply:
ah ignore that, are you intending to add a new `school_project` for each project, as I think we'll need that? I think the logic needs adjusting so that it's checking if `school_project.blank?` and `school_id.present?`, and creating a new record if so?

Reply:
I think I just added this in for the sake of completeness, in case we ever roll these migrations back for some reason